site/content/en/docs/Installation/Creating Cluster/eks.md

```diff
@@ -41,6 +41,12 @@ For Agones to work correctly, we need to allow UDP traffic to pass through to ou
 * Select **Inbound Rules**
 * **Edit Rules** to add a new **Custom UDP Rule** with a 7000-8000 port range and an appropriate **Source** CIDR range (`0.0.0.0/0` allows all traffic)
 
+## Use with custom CNI plugins
+
+To ensure compatibility with custom CNI plugins (such as Calico or Cilium), configure extensions application to use `hostNetwork` for proper functionality.
```

Comment:
```suggestion
To ensure compatibility with custom CNI plugins (such as Calico or [Cilium](https://docs.cilium.io/en/stable/installation/k8s-install-helm/)), configure extensions application to use `hostNetwork` for proper functionality.
```

Although, I'd love a way to link to the EKS docs tab, since it specifically references using hostNetwork?